launchpad-to-stargate/tutorial/02-module-migration.md

```diff
@@ -0,0 +1,688 @@
+
+The most important file to note is the modification we made in the `launchpad/pofe/x/pofe/client/cli/txClaim.go` file:
+
+```go
+// launchpad
+
+package cli
+
+import (
+	"bufio"
+	"crypto/sha256"
+	"encoding/hex"
+	"io/ioutil"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	"github.com/user/pofe/x/pofe/types"
+)
+
+// CLI transaction command to create a claim
+func GetCmdCreateClaim(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "create-claim [path-to-file]",
+		Short: "Creates a new claim from a path to a file",
+		Args:  cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// accept a filepath, read the file, and hash it
+			hasher := sha256.New()
+			s, _ := ioutil.ReadFile(args[0])
+			hasher.Write(s)
+			argsProof := hex.EncodeToString(hasher.Sum(nil))
+
+			// automatically scaffolded by `starport type`
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			inBuf := bufio.NewReader(cmd.InOrStdin())
+			txBldr := auth.NewTxBuilderFromCLI(inBuf).WithTxEncoder(utils.GetTxEncoder(cdc))
+			msg := types.NewMsgCreateClaim(cliCtx.GetFromAddress(), string(argsProof))
+			err := msg.ValidateBasic()
+			if err != nil {
+				return err
+			}
+			return utils.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
+		},
+	}
+}
+
+func GetCmdSetClaim(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "set-claim [id]  [proof]",
+		Short: "Set a new claim",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			id := args[0]
+			argsProof := string(args[1])
+
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			inBuf := bufio.NewReader(cmd.InOrStdin())
+			txBldr := auth.NewTxBuilderFromCLI(inBuf).WithTxEncoder(utils.GetTxEncoder(cdc))
+			msg := types.NewMsgSetClaim(cliCtx.GetFromAddress(), id, string(argsProof))
+			err := msg.ValidateBasic()
+			if err != nil {
+				return err
+			}
+			return utils.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
+		},
+	}
+}
+
+func GetCmdDeleteClaim(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "delete-claim [id]",
+		Short: "Delete a new claim by ID",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			inBuf := bufio.NewReader(cmd.InOrStdin())
+			txBldr := auth.NewTxBuilderFromCLI(inBuf).WithTxEncoder(utils.GetTxEncoder(cdc))
+
+			msg := types.NewMsgDeleteClaim(args[0], cliCtx.GetFromAddress())
+			err := msg.ValidateBasic()
+			if err != nil {
+				return err
+			}
+			return utils.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
+		},
+	}
+}
+```
+
+As well as boilerplate `txClaim.go` file in our Stargate application:
```

Comment:
Seems maybe obvious but indicating the `x/pofe/client/cli/txClaim.go` path here might be helpful.